### PR TITLE
Add MM - Millennium Media

### PR DESCRIPTION
**Pull request by japierce1028**
Opt OUT

### CODE REVIEW

File: src/main/data/studios.json [hunk 1]
```diff
@@ -708,6 +708,11 @@
     "code": "MI",
     "description": "METRO INTERNATIONAL"
   },
+  {
+    "code": "MM",
+    "description": "Millennium Media",
+    "url": "http://millennium-media.net/"
```
Comment (palemieux) [inline src/main/data/studios.json:714]:
```suggestion
    "url": "https://millennium-media.net/"
```

**Review by palemieux — CHANGES_REQUESTED** [review]

**Comment by japierce1028** [comment]
Agreed. https is better.

**Comment by japierce1028** [comment]
Oops, Must have done something wrong...

File: src/main/data/studios.json (hunk 1 above)
Comment (palemieux) [inline src/main/data/studios.json:714]:
```suggestion
    "url": "https://millennium-media.net/"
```

**Review by palemieux — CHANGES_REQUESTED** [review]

**Comment by palemieux** [comment]
> Oops, Must have done something wrong...

You have to click "commit change".

**Review by palemieux — APPROVED** [review]

**Pull request merged**